Module 2/Module 2.1 Assignment.md

```diff
@@ -1,20 +1,17 @@
 # Module 2.1 Assignment
 
-In this assignment you will create a remote on your computer with this repository. In [Module 2.1](https://youtu.be/R2bLo-KiYlU)
-you were given an introduction on how to create a remote from scratch, but what happens if you want to create a remote for a
-repository that already exists. Now, if a repository already exists, and it already has code that you want, 
-you can actually copy it to your local machine using the command `git clone` followed by the Repository URL.
+In this lesson, you learned how to create a remote repository from scratch. In this assignment, you’ll learn how to clone a repository on GitHub to your local machine, allowing you to manage and edit files locally. To complete this assignment, follow the instructions below:
 
-Lets do this step by step:
+1. On GitHub, navigate to the main page of this repository. Click the green button that says `Code` then copy the URL by clicking the gray clipboard icon.  
```

Comment:
I'm concerned here that someone might start at GitHub.com instead of their own repository and get lost, and I think this edit might mitigate that

```suggestion
1. Navigate to the main page of this repository on GitHub. Click the green button that says `Code` then copy the URL by clicking the gray clipboard icon.  
```

Reply:
Good call!